src/main/java/org/cbio/gdcpipeline/processor/MutationDataProcessor.java

```diff
@@ -26,4 +30,25 @@ private String extractSampleId(String record) {
         }
         return record;
     }
+
+    private String normalizeChromosome(String chromosome){
```

Comment:
It looks like you aren't doing any chromosome normalization before calling the annotate method. I think it makes sense to change things like `CHRX` etc. to `X` before calling annotate record, and `23` and `24` to `X` and `Y` respectively to make annotation more robust (kind of going in the reverse direction, as genome nexus/VEP works with with the letters, not the numbers). Then you could leave this `normalizeChromosome` method here as it is. This would go in the `org.cbioportal.annotator.internal.GenomeNexusImpl` class. Let me know if this doesn't make sense.

Reply:
I made a new commit on genome nexus PR. Please let me know if it looks good